getstarted.md

```diff
@@ -0,0 +1,51 @@
+## Completing Juju Shell setup
+
+The Juju Shell requires a valid SSL certificate and it supports 
+[Let's Encrypt](https://letsencrypt.org/) to auto load a certificate once the
+DNS is set. Visit your domain provider or DNS tool for your domain and set up
+a subdomain for your Juju Shell. You'll need the IP address of the exposed
+Juju Shell unit.
+
+    $ juju expose jujushell
+    $ juju status jujushell
+
+    Model    Controller  Cloud/Region     Version  SLA
+    default  jujushell   google/us-east1  2.3.2    unsupported
+
+    App        Version  Status   Scale  Charm      Store       Rev  OS      Notes
+    jujushell           waiting    0/1  jujushell  jujucharms    3  ubuntu
+    exposed
+
+    Unit         Workload  Agent       Machine  Public address  Ports  Message
+    jujushell/0  waiting   allocating  0        35.196.154.94          waiting for
+    machine
+
+    Machine  State    DNS            Inst id        Series  AZ          Message
+    0        pending  35.196.154.94  juju-e62c69-0  xenial  us-east1-b  RUNNING
+
+You want to pull the `Public Address` from the status of the Juju Shell and
+set that to your DNS record.
+
+    jujushell.mydomain.com.        IN      A       35.196.154.94
```

Comment:
Maybe specify that jujushell.* is just an example, that it could be anything?

Reply:
sure thing